### PR TITLE
chore: add Arbitrary / util tests

### PR DESCRIPTION
**Pull request by jacob-alford**
Brings coverage back up to 100%

### CODE REVIEW

File: src/internal/util.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import fc, { Arbitrary } from 'fast-check'
 
+/* istanbul ignore next */
 /** @internal */
 export const base64Encode = (s: string): string =>
   Buffer ? Buffer.from(s).toString('base64') : btoa(s)
 
+/* istanbul ignore next */
```
Comment (jacob-alford) [inline src/internal/util.ts:8]: I thought I covered all branch cases, but `jest` disagreed,  so instead I used `istanbul ignore next`

Comment (skeate) [reply src/internal/util.ts:8]: Oops. The default case is unreachable and Istanbul doesn't know it

Comment (jacob-alford) [reply src/internal/util.ts:8]: Oh I see, that makes sense.  So we should be able to remove the explicit final case, and remove the istanbul-ignore-next?

**Review by newswim — APPROVED** [review]
Lgtm!

**Review by newswim — APPROVED** [review]
👨‍🍳

**Pull request merged**